classes/class-plugin.php

```diff
@@ -18,7 +18,7 @@ class Plugin {
 	 *
 	 * @const string
 	 */
-	const VERSION = '3.10.0';
+	const VERSION = '3.10.1';
```

Comment:
<!-- simple tasks that must be done -->
**🧹 chore:** Let's not bump the version number just yet. This update will be part of `4.0.1` release and we're going to set the plugin version there, just before releasing.

```suggestion
	const VERSION = '3.10.0';
```